_posts/2020-08-13-convertextract-app.md

```diff
@@ -0,0 +1,98 @@
+---
+layout: post
+title:  "How to use the new Convertextract application for 'quality control' of ELAN annotations"
+author: fineen
+categories: [ Tutorial, Blog ]
+tags: [ basic ]
+image: assets/images/gui.png
+description: "How to install and use the new Convertextract app for 'quality control' purposes"
+featured: false
+hidden: false
+---
+
+# TL;DR
+
+Have you ever wanted to *NOT* spend hours tediously checking that **k + '** is written as **k̓** and not **k'**?
+If you said **YES!**, *Convertextract* is the app for you. With minimal technical knowledge, you can now systemically make your ELAN annotations consistent.
+
+# What you need to know to understand this post
+
+I assume that you know some background about the [g2p library](https://github.com/roedoejet/g2p), but if your language already has the desired template for conversion (mapping), then you don't need to worry too much about this.
+
+# Who is involved with this project?
+
+- Kwak̓wala Corpus Collection group
+
+	Sara Child [saratlilinukw@gmail.com](mailto:saratlilinukw@gmail.com)
+	Daisy Rosenblum [daisy.rosenblum@ubc.ca](mailto:daisy.rosenblum@ubc.ca)
+	Caroline Running Wolf [caroline.oldcoyote@gmail.com](mailto:caroline.oldcoyote@gmail.com)
+
+- App developer: [Aidan Pine](aidanpine.ca)
+
+- Support for adding mappings/parsers: [Fineen](fineen.davis@gmail.com)
+
+
+# What is needed to replicate the content in the post?
+
+- [g2p](https://github.com/roedoejet/g2p) mapping of the desired conversions
+- Language text to be converted
+- *Convertextract* application (read the post for installation!) 
+
+# What are the motivations behind this technology?
+
+As a Student Intern on the NRC's [Indigenous Language Technology (ILT) project](https://nrc.canada.ca/en/research-development/research-collaboration/programs/canadian-indigenous-languages-technology-project), I was approached by the Kwak̓wala Corpus Collection group to help create a systemic way to streamline the quality control process for their ELAN annotation data. Having many different people with many different orthographic conventions all working on annotating Kwak̓wala language data had resulted in inconsistencies.
```

Comment:
* What aspect *specifically* are you helping with the quality control? 
* What is an orthographic convention? 
* If my language has different orthographic conventions, is this the technology that I could/should use?